README.md

```diff
@@ -0,0 +1,58 @@
+# Clear Containers Tests Repository
+
+This repository contains the test suite for the Clear Containers 3.0 project,
+including functional and integration tests with Docker\*, Kubernetes\* and OpenShift\*.
+
+An environment with Clear Containers properly setup is needed in order to execute 
+these tests. If you require to setup Clear Containers, please refer to the:
+[Installation Guides](https://github.com/clearcontainers/runtime/tree/master/docs)
+
+## Functional tests
+Execute:
+```
+	$ sudo -E PATH=$PATH make functional
+```
+## Docker Integration tests
+
+Execute:
+```
+	$ sudo -E PATH=$PATH make integration
+```
+
+## Functional and Docker integration tests
+Execute:
+```
+	$ sudo -E PATH=$PATH make check
+```
+
+## Environment variables
+
+By default, these tests use the version of `cc-runtime` set in the environment
+variable `RUNTIME`, but you can easily change it. For example:
+```
+	$ RUNTIME="/usr/local/bin/cc-runtime" make functional
+```
+In the above example the version installed in `/usr/local/bin` of the Runtime is used.
```

Comment:
This is confusing as the variable is called runtime and we are talking about the runtime version. I suggest we clarify:
In the above example, the version of runtime installed in `/usr/loval/bin` is being used.